server/api/impl/admin/list.go

```diff
@@ -4,28 +4,64 @@
 package admin
 
 import (
+	"strings"
+
 	"github.com/mattermost/mattermost-plugin-apps/apps"
-	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
 )
 
-func (adm *Admin) ListApps() ([]*apps.App, md.MD, error) {
-	apps := adm.store.App().GetAll()
-	if len(apps) == 0 {
-		return nil, "no apps installed", nil
-	}
+func (adm *Admin) GetApp(appID apps.AppID) (*apps.App, error) {
+	return adm.store.App().Get(appID)
+}
+
+func (adm *Admin) GetManifest(appID apps.AppID) (*apps.Manifest, error) {
+	return adm.store.Manifest().Get(appID)
+}
+
+func (adm *Admin) ListInstalledApps() map[apps.AppID]*apps.App {
+	return adm.store.App().AsMap()
+}
+
+func (adm *Admin) ListMarketplaceApps(filter string) map[apps.AppID]*apps.MarketplaceApp {
```

Comment:
Is this the proper name since manifests "They generally come from the marketplace, and can also be installed as overrides by developers."?